src/main/java/rx/plugins/RxJavaErrorHandler.java

```diff
@@ -44,4 +48,47 @@ public void handleError(Throwable e) {
         // do nothing by default
     }
 
+    protected static final String ERROR_IN_RENDERING_SUFFIX = ".errorRendering";
+
+    /**
+     * Receives items causing {@link OnErrorThrowable.OnNextValue} and gives a chance to choose the String
+     * representation of the item in the OnNextValue stacktrace rendering. Returns null if this type of item
+     * is not managed and should use default rendering.
+     * <p>
+     * Note that primitive types are always rendered as their toString() value.
+     * <p>
+     * If a {@code Throwable} is caught when rendering, this will fallback to the item's classname suffixed by
+     * {@value #ERROR_IN_RENDERING_SUFFIX}.
+     *
+     * @param item the last emitted item, that caused the exception wrapped in {@link OnErrorThrowable.OnNextValue}.
+     * @return a short {@link String} representation of the item if one is known for its type, or null for default.
+     */
+    @Experimental
+    public final String handleOnNextValueRendering(Object item) {
+        try {
+            return render(item);
+        } catch (Throwable t) {
+            return item.getClass().getName() + ERROR_IN_RENDERING_SUFFIX;
```

Comment:
Swallowing all `Throwable`s may cause some issue. I suggest the following approach:
```Java
} catch (InterruptedException e) {
    Thread.currentThread().interrupt();
} catch (Throwable t) {
    Exceptions.throwIfFatal(t);    
}
return item.getClass().getName() + ERROR_IN_RENDERING_SUFFIX;
```